tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -2049,6 +2152,15 @@ bool CUDABlas::DoBlasGemmWithAlgorithmImpl(
     return false;
   }
 
+#if CUDA_VERSION >= 9000
+  cublasGemmAlgo_t cublas_algo = static_cast<cublasGemmAlgo_t>(algorithm);
+  if (cublas_algo >= CUBLAS_GEMM_DFALT_TENSOR_OP &&
```

Comment:
Why not using "CUBLAS_GEMM_DEFAULT_TENSOR_OP".

Also I'm not a fan of comparing enums. Could you split "cublas_algo >= CUBLAS_GEMM_DFALT_TENSOR_OP" into a function? In fact, I'd prefer to return them one by one... So future changes have to update them.

Reply:
Changed to CUBLAS_GEMM_DEFAULT_TENSOR_OP and moved inequality check to a separate function.